File: app/src/main/java/com/BlizzardArmory/warcraft/equipment/Armor.java
package com.BlizzardArmory.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Armor {

    @SerializedName("value")
    @Expose
    private int value;
    @SerializedName("display_string")
    @Expose
    private String displayString;
    @SerializedName("display")
    @Expose
    private Display display;

    public Display getDisplay() {
        return display;
    }

    public void setDisplay(Display display) {
        this.display = display;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    public String getDisplayString() {
        return displayString;
    }

    public void setDisplayString(String displayString) {
        this.displayString = displayString;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Armor.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Armor.
 */
@Keep
data class Armor(

    @SerializedName("value")
    var value: Int,

    @SerializedName("display_string")
    var displayString: String?,

    @SerializedName("display")
    var display: Display

)